projects/netgrif-components/src/lib/side-menu/content-components/user-impersonate/user-impersonate.component.spec.ts

```diff
@@ -0,0 +1,60 @@
+// import {ComponentFixture, TestBed} from '@angular/core/testing';
```

Comment:
why is this file commented out?

Reply:
this component is heavily inspired by load-filter.component. The test for the other component is also commented out with a message saying "gave-up on load filter component test" and as I also had no luck getting this to work, I just left it commented out in hopes it will get resolved in the future

Reply:
I would add at least TODOs to both files, because I was unaware that they existed...
It would be even better to create a new Jira Issue, so that this can be solved